README.md

```diff
@@ -1,76 +1,132 @@
+| [Express](http://expressjs.com/) | 4+ | `use('express')` | |
+| [GraphQL](https://github.com/graphql/graphql-js) | 0.10+ | `use('graphql')` | |
+| [hapi](https://hapijs.com/) | 2+ | `use('hapi')` | |
+| [http/https](https://nodejs.org/api/http.html) | Supported Node | `use('http')`, `use('https')` | |
+| [ioredis](https://github.com/luin/ioredis)| 2+ | `use('ioredis')` | |
+| [Koa](https://koajs.com/) | 2+ | `use('koa')` | |
+| [Knex](https://knexjs.org/) | 0.10+ | `use('bluebird'); use('knex')` | Depends on bluebird instrumentation |
+| [Memcached](https://github.com/3rd-Eden/memcached) | 2.2+ | `use('memcached')` | |
+| [MongoDB-Core](https://github.com/mongodb-js/mongodb-core) | 2+ | `use('mongodb-core')` | |
+| [mysql](https://github.com/mysqljs/mysql) | 2+ | `use('mysql')` | |
+| [MySQL2](https://github.com/sidorares/node-mysql2) | 1+ | `use('mysql2')` | |
+| [Nest](https://nestjs.com/) | 4.x - 6.x | `use('nest')` | |
+| [Net](https://nodejs.org/api/net.html) | Supported Node | `use('net')` | |
+| [node-postgres](https://github.com/brianc/node-postgres) | 4+ | `use('pg')` | |
+| [Pino](http://getpino.io/#/) | 2+ | `use('pino')` | |
+| [Q](https://github.com/kriskowal/q) | 1+ | `use('q')` | |
+| [Redis](https://github.com/NodeRedis/node_redis) | 0.12+ | `use('redis')` | |
+| [restify](http://restify.com/) | 3+ | `use('restify')` | |
+| [Sails](https://sailsjs.com) | 1+ | `use('sails')` | |
+| [Socket.IO](https://socket.io) | 1.2+ | `use('socket.io')` | |
+| [when.js](https://github.com/cujojs/when) | 3+ | `use('when')` | |
+| [winston](https://github.com/winstonjs/winston) | 1+ | `use('winston')` | |
+
+## Configure the SignalFx Tracing Library for JavaScript
+
+Send traces from your Java application to a local or remote Smart Agent,
+OpenTelemetry Collector, or SignalFx ingest endpoint.
+
+### Configuration values
+
+Configure these options as parameters for the `init()` method or as environment variables.
+
+| Config        | Environment Variable         | Default   | Description |
+| ------------- | ---------------------------- | --------- | ----------- |
+| service       | SIGNALFX_SERVICE_NAME        | unnamed-nodejs-service | The service name to be used for this program. |
+| url           | SIGNALFX_ENDPOINT_URL        | http://localhost:9080/v1/trace | The url of the Agent or Gateway to which the tracer will submit traces. |
+| accessToken   | SIGNALFX_ACCESS_TOKEN        |           | The optional organization access token for trace submission requests |
+| enabled       | SIGNALFX_TRACING_ENABLED     | true      | Whether to enable the tracer. |
+| debug         | SIGNALFX_TRACING_DEBUG       | false     | Enable debug logging in the tracer. |
+| logInjection  | SIGNALFX_LOGS_INJECTION      | false     | Enable automatic injection of trace IDs in logs for supported logging libraries. |
+| tags          |                              | {}        | Set global tags that should be applied to all spans. |
```

Comment:
This was updated and should reflect what's in master (SIGNAFX_SPAN_TAGS and format note).